1_fetch/src/get_feature_vars.R

```diff
@@ -112,11 +159,34 @@ get_sb_data_log <- function(sb_var_ids, file_out) {
   
   write_csv(sb_log, file_out)
   return(file_out)
+}
+
+
+prep_comid_conus <- function(nhd_conus_gdb, attrib_to_keep, ftype_to_keep, outdir) {
+  
+  #'@description Selects comids of interest for conus-wide predictions
+  #'
+  #'@param nhd_conus_gdb target from p1_nhd_conus_gdb (raw unzipped nhd geodatabase)
+  #'@param attrib_to_keep character list of attributes to keep (columns in nhd_conus_gdb)
+  #'@param ftype_to_keep character list of flowline types to retain (of the following:
+  #'ArtificialPath, StreamRiver, Connector, CanalDitch, Coastline, Pipeline)
+  #'@param outdir filepath to save final csv
+  #'
+  #'@return non-tidal conus-wide comids of specified ftype(s)
+  #'with specified attributes retained from geodatabase (class 'sf')
   
+  nhd_full <- st_read(nhd_conus_gdb, layer = 'NHDFlowline_Network')
+  comid_conus <- nhd_full %>%
+    select(all_of(attrib_to_keep)) %>%
+    filter(FTYPE %in% all_of(ftype_to_keep)) %>%
+    filter(Tidal == 0)
+  
+  return(comid_conus)
```

Comment:
let's transform to EPSG 4326 coordinate system before returning this file. That's what we use for g2 data. May also need to remove the ZM with st_zm().